app/rpc/RPC.ts

```diff
@@ -792,6 +792,14 @@ export default class RPC {
       if (fullRescan) {
         // clean the transaction list before.
         this.fnSetTransactionsList([]);
+        this.fnSetTotalBalance({
+          orchardBal: 0,
+          privateBal: 0,
+          transparentBal: 0,
+          spendableOrchard: 0,
+          spendablePrivate: 0,
+          total: 0,
+        });
```

Comment:
seeing this has got me thinking....
...if we keep track of "balance accounting" in zingo-mobile we are at risk of balance bugs...
would it not be better to handle balance by:

calling balances directly from rust and letting zingolib calculate by summing our note sets etc.
OR
always calculating balance in zingo-mobile by summing note set returned from rust
maybe this is how zingo-mobile already works and this 'reset to 0' is just to update the UI quickly?
but i think that when the transanction list is cleared, this should reset the balances to 0 because there are no notes to sum. instead of starting from 0 and adding / subtracting when notes are received or spent

Reply:
in this case the `rescan` process can take some time to start & the periodic process asking for transactions & balance can be not synchronous with that... because of that, I prefer to clean both manually in UI just before the `rescan` is starting... in fact before this fix, first disappear the txs and after a while the balance is setting to cero. 
This for the user was weird.